instrumentation/runtime-telemetry-jfr/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/runtimetelemetryjfr/RuntimeMetricsInstallerJfr.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.javaagent.runtimetelemetryjfr;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.runtimetelemetryjfr.JfrTelemetry;
+import io.opentelemetry.javaagent.extension.AgentListener;
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+
+/** An {@link AgentListener} that enables runtime metrics during agent startup. */
+@AutoService(AgentListener.class)
+public class RuntimeMetricsInstallerJfr implements AgentListener {
+
+  @Override
+  public void afterAgent(AutoConfiguredOpenTelemetrySdk autoConfiguredSdk) {
+    ConfigProperties config = autoConfiguredSdk.getConfig();
+
+    boolean defaultEnabled = config.getBoolean("otel.instrumentation.common.default-enabled", true);
```

Comment:
Does this say that jfr instrumentation is enabled by default?

Reply:
Yes, I thought that might be a good choice because its similar to the JMX gatherer, which is also collecting metrics. And by default we are only collecting metrics not already covered by JMX.

Reply:
> And by default we are only collecting metrics not already covered by JMX.

True.. although the metrics it produces aren't part of the semantic conventions (yet). Wondering if that influences our decision.

Generally, I think they contain useful data, but need just a bit of adjustment to match the [metric semantic conventions guidelines](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#general-guidelines). I.e. they are likely to change before being added.

Reply:
Hmm I see, yeah I think maybe its better to disable them by default. Then once its decided how the semantic conventions can be updated, they can be enabled by default.

Reply:
Also discussed at the [March 2 2023 SIG](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/edit#heading=h.7khhmsx7w18m).  Decided that the new metrics should not be enabled by default until the semantic conventions can be updated.